.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+    "singleQuote": true,
+    "trailingComma": "all",
+    "printWidth": 120,
+    "semi": true
```

Comment:
True is de default waarde voor deze instelling, dus niet nodig deze expliciet op te nemen.

Reply:
dit zijn instellingen uit slimme apparaten

Reply:
als je die wil aanpassen goed